apiary.apib

```diff
@@ -374,6 +374,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added lost_reason to the response of to `deals.list`.
```

Comment:
Other fields in the `Latest` section have backtick around them. 
For consistency `lost_reason` should have too.